rstest_macros/src/render/mod.rs

```diff
@@ -57,14 +59,18 @@ pub(crate) fn single(mut test: ItemFn, info: RsTestInfo) -> TokenStream {
 }
 
 pub(crate) fn parametrize(test: ItemFn, info: RsTestInfo) -> TokenStream {
-    let RsTestInfo { data, attributes } = info;
+    let RsTestInfo {
+        data,
+        attributes,
+        await_args,
+    } = info;
     let resolver_fixtures = resolver::fixtures::get(data.fixtures());
 
     let rendered_cases = cases_data(&data, test.sig.ident.span())
         .map(|(name, attrs, resolver)| {
             TestCaseRender::new(name, attrs, (resolver, &resolver_fixtures))
         })
-        .map(|case| case.render(&test, &attributes))
+        .map(|case| case.render(&test, &await_args, &attributes))
         .collect();
```

Comment:
I'm quite sure that we can avoid to spread `await_args` in all functions signature... I think that `test` and `attributes` can contains all information. I we start to add a new argument for each feature the code will become a mess (more than it already is :) ).